ama-sample.conf

```diff
@@ -1,17 +1,24 @@
 {
+  #required
   pg => 'postgresql://mojo:mojo@/test1',
+  #recommended
   secrets => ['mojo'],
+  #leave blank to remove login functionality
```

Comment:
Leave blank...  Does that mean to not define the key, or to define it but leave its value empty?  That is:

`oauth2 => {},`

Or do not provide an oauth2 key whatsoever?  Same comment for the other comments where you say "leave blank".